cli/setup.py

```diff
@@ -20,8 +20,8 @@
     install_requires=[
         'click',
         'requests',
-        'pyyaml',
-        'cryptography'
+        'lark-parser',
+        'pyyaml'
```

Comment:
The cryptography module is not present by default in the python 3.x versions. So I think we should keep it here, to keep our codebase compatible with both python 2.x and 3.x